test/GlobStream.js

```diff
@@ -6,6 +6,16 @@ var concat = miss.concat;
 var pipe = miss.pipe;
 
 describe('GlobStream', function() {
+  it('emits an error if it is not first in the stream', function(done) {
+    var gs1 = stream('./fixtures/whatsgoingon/**/*.txt', [], { cwd: __dirname, });
+    var gs2 = stream('./fixtures/whatsgoingon/**/*.txt', [], { cwd: __dirname, });
+
+    pipe([
+      gs1,
+      gs2,
+    ]);
+  });
```

Comment:
Could actually use some help on this one, I'm not positive on how to catch the error thrown by `gs2`. So far tried throwing event listeners for `'error'` on both of the pipe, both of the streams (just in case), and as a last resort a try/catch around the whole thing to no avail. Tried digging in to readable streams' `_stream_readable.js` but thought you guys might be able to shed some light on it as well. Thanks!